tests/unit/testsuite/Bolt/Boltpay/OrderHelper.php

```diff
@@ -145,7 +145,11 @@ public static function deleteDummyOrderByIncrementId($incrementId)
      */
     public static function deleteDummyOrder(Mage_Sales_Model_Order $order)
     {
-        self::deleteDummyOrderByIncrementId($order->getIncrementId());
-
+        if (!empty($order)) {
+            $id = $order->getIncrementId();
+            if (!empty($id)) {
+                self::deleteDummyOrderByIncrementId($id);
+            }
+        }
```

Comment:
This appears to be unrelated to the subject of the PR and should be in it's own PR.